app/main/views/frameworks.py

```diff
@@ -11,3 +14,42 @@ def list_frameworks():
     return jsonify(
         frameworks=[f.serialize() for f in frameworks]
     )
+
+
+@main.route('/frameworks/<string:framework_slug>/stats', methods=['GET'])
+def get_framework_stats(framework_slug):
+
+    framework = Framework.query.filter(
+        Framework.slug == framework_slug
+    ).first()
+
+    if not framework:
+        abort(404, "'{}' is not a framework".format(framework.slug))
+
+    seven_days_ago = datetime.datetime.utcnow() + datetime.timedelta(-7)
+    lot_column = DraftService.data['lot'].cast(String).label('lot')
+
+    return str({
+        'services_by_status': dict(db.session.query(
+            DraftService.status, func.count(DraftService.status)
+        ).group_by(
+            DraftService.status
+        ).filter(
+            DraftService.framework_id == framework.id
+        )),
```

Comment:
It's possible to get service counts for each lot and status combination with a single query and then sum it in the frontend if we want. Since lots have different question counts I think "complete" split will be different from total counts.